### PR TITLE
Updated client to output hist in date order

### PR DESCRIPTION
**Pull request by robhudson**
This is wanted for our charting library to do less fiddling with the returned data in Javascript.

### CODE REVIEW

File: monolith/client/__init__.py [hunk 1]
```diff
@@ -159,18 +159,17 @@ def __call__(self, field, start, end, interval=DAY, strict_range=False,
             else:
                 count = entry['count']
 
-            if date_ not in dates:
-                dates.add(date_)
+            if date_ not in counts:
+                counts[date_] = count
```
Comment (almet) [inline monolith/client/__init__.py]: if `date` is already in `counts`, shouldn't we sum the counts?

**Comment by almet** [comment]
Sounds good to me.

File: monolith/client/__init__.py (hunk 1 above)
Comment (robhudson) [inline monolith/client/__init__.py]: The summing of dates happens on import as long as the dimensions are the same. Elasticsearch isn't going to (or shouldn't ever) return two counts for the same date, though.

Comment (almet) [inline monolith/client/__init__.py]: okay, so this is just a paranoid check? :)

Comment (robhudson) [inline monolith/client/__init__.py]: Heh. I suppose? It was there originally so I guess I didn't want to change it too drastically. But if we agree it's not needed I could refactor and test it. Let me do that.

File: monolith/client/__init__.py [hunk 2]
```diff
@@ -158,19 +158,16 @@ def __call__(self, field, start, end, interval=DAY, strict_range=False,
                 count = entry['total']
             else:
                 count = entry['count']
+            counts[date_] = count
```
Comment (robhudson) [inline monolith/client/__init__.py:161]: Updated to remove paranoia. :)

**Pull request merged**